bower.json

```diff
@@ -22,8 +22,8 @@
   "license": "https://github.com/cfpb/cf-buttons/blob/gh-pages/TERMS.md",
   "main": "src/cf-buttons.less",
   "dependencies": {
-    "cf-core": "git://github.com/cfpb/cf-core.git",
-    "cf-icons": "git://github.com/cfpb/cf-icons.git"
+    "cf-core": "latest",
+    "cf-icons": "latest"
```

Comment:
Yeah, that's how we've been targeting them. When you provide a git endpoint with no tag it pulls the default branch, which would be the latest and greatest.